### PR TITLE
feat: Support lerna scopes in npm start

### PR DESCRIPTION
**Pull request by bmingles**
Support lerna scopes in npm start

### CODE REVIEW

File: .prettierrc
Comment (mofojed) [inline .prettierrc:1]: Shouldn't need this - we already have the prettier config specified in package.json

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+plugins/json/test/*.json
```
Comment (mofojed) [inline .prettierignore:1]: Why do we need to ignore these files?

Comment (bmingles) [reply .prettierignore:1]: Formatting breaks tests. I assume they are formatted to match the diff exactly

File: .prettierrc
Comment (bmingles) [reply .prettierrc:1]: deleted

**Review by mofojed — APPROVED** [review]

**Pull request merged**